pkg/command/command_test.go

```diff
@@ -55,7 +55,6 @@ func TestFailurePipeWrongArgument(t *testing.T) {
 	require.Nil(t, err)
 	require.False(t, res.Success())
 	require.Empty(t, res.Output())
-	require.Contains(t, res.Error(), "unrecognized option")
```

Comment:
nit: Wouldn't ...
```go
require.NotEmpty(t, res.Error())
```
... be closer to what you want to test?

Reply:
Makes sense, changed it as suggested :+1: